CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ message(STATUS "Building fastscapelib v${PROJECT_VERSION}")
 
 set(FASTSCAPELIB_INCLUDE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/include)
 
+OPTION(FS_USE_XTENSOR_CONTAINERS "Configure fastscapelib to allow xtensor containers" ON)
```

Comment:
With this PR is it already possible to use fastscapelib without xtensor? I.e., if I set it to `OFF` will it work (tests, python bindings, downstream C++ project, etc.) without any other extra care?

If not, I think that it will be better (less confusing) to expose such option later (alongside with other options for alternative containers such as eigen).

Reply:
agree! I added this but finally figured out that it would be hard to make it a soft dependency for now. I'll remove it